### DISCUSSION TITLE
TestFlight beta 1.27.10 with Photo Collection enhancements

### DISCUSSION THREAD

**Discussion by SimonPickup** [MobiusSync/MobiusSync#194] [Beta]
If you are not already a beta tester, join here:

For Möbius Sync (Freemium), register here: https://testflight.apple.com/join/sBZ8ZuCV
For Möbius Sync Pro, register here: https://testflight.apple.com/join/FokR8tfx

First build including Syncthing 1.27.10 will be available shortly. Subscribe to this thread for updates, and please report your feedback here.

**Comment by alx-xlx** [comment]
What is the difference between these two, I have the Mobius Sync, but I think I paid for something in it, I don't exactly remember. Does Mobius Sync Pro has better features ?? its really confusing

**Reply by SimonPickup** [reply]
Please see the FAQ.

**Reply by alx-xlx** [reply]
> Please see the FAQ.

hey I read the FAQ and it says "They are identical in terms of functionality, once paid. They merely represent two purchasing methods."
However when I see the Mobius Sync it is on syncthing 1.27.10 and the Mobius Sync Pro is on syncthing 1.28.xx

I paid for the Mobius Sync and I am not getting the Pro features, can you please upgrade me to the pro tier ?

**Comment by rotbaer-peter** [comment]
Thank you, I just installed the Test Flight Build and bug #177 seems to be solved. If it returns, I will give you feedback!

**Comment by rotbaer-peter** [comment]
The flickering issue bug #177 is still there on both my devices (iPad pro 2024, iPhone 14pro). App crashed after closing and reopening.

**Comment by SimonPickup** [comment]
New TestFlight build 202410100810 is out:

New this build:
- Bug fix: #193
- #197

Previous builds:

#191

Adds Photo Collection folder options:
- #178
- #180
- #181
- #192
- #198

**Comment by SimonPickup** [comment]
This is now released publicly to the App Store.
